benchmarks/smallmatvec/manual_versions/openmp/scalability.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
```

Comment:
Please add a comment describing what this script does.

Reply:
Done